cmd/action/apply.go

```diff
@@ -65,10 +67,10 @@ func CmdApplyRun(cmd *cobra.Command, args []string) {
 	}
 	d, err := defaultMux.OpenAtlas(ApplyFlags.DSN)
 	cobra.CheckErr(err)
-	applyRun(d, ApplyFlags.DSN, ApplyFlags.File, ApplyFlags.DryRun)
+	applyRun(d, ApplyFlags.DSN, ApplyFlags.File, ApplyFlags.DryRun, ApplyFlags.AutoApprove)
 }
 
-func applyRun(d *Driver, dsn string, file string, dryRun bool) {
+func applyRun(d *Driver, dsn string, file string, dryRun bool, autoApprove bool) {
```

Comment:
@rotemtam should we introduce a small config struct here?

Reply:
Yes, but can be done in additional PR.